omisego/elixir-omg
# fix: remove redundant env vars listing in docs and add make api_specs

## Overview

Remove redundant env vars listing from docs and add `make api_specs` to generate operator, watcher and watcher_info swagger specs in one go.

## Changes

- Remove redundant env var docs
- Add `make api_specs` to make me less stressed generating the api specs

## Testing

- See docs/deployment_configuration.md 
- Run `make api_specs`
